utils/src/main/scala/za/co/absa/enceladus/utils/types/TypedStructField.scala

```diff
@@ -178,6 +184,22 @@ object TypedStructField {
     }
   }
 
+  // BinaryTypeStructField
+  final class BinaryTypeStructField private[TypedStructField](structField: StructField)
+                                                             (implicit defaults: Defaults)
+    extends TypedStructFieldTagged[Array[Byte]](structField) {
+
+    // used to convert the default value from metadata's [[MetadataKeys.DefaultValue]]
+    override protected def convertString(string: String): Try[Array[Byte]] = {
+      //binary's default value is always expected to be in base64
+      Try(Base64.getDecoder.decode(string))
```

Comment:
This will always use the base64 decoder, which is not desired, IMO.
It should be consittent with the encoding, like in other cases. But glad to discuss.

Reply:
I thought we wanted it that way. OK, redone to respect the encoding, too.